scripts/upload_release_assets.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# This script has the following argument:
+#     - releaseID (mandatory)
+# ./upload_release_assets.sh 12345678
+
+set -x
+
+RELEASE_ID=${1}
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
```

Comment:
Can we shorten it and use `set -e`?

Reply:
We consistently use this code across plethora of other scripts. Do you want to change all the scripts?

Reply:
No need to do it now. Let's discuss it after the PR is merged.